components/mixins/FixedTop.js

```diff
@@ -17,15 +18,20 @@ export default {
     isScrollingDown () {
       return this.currScrollY > this.prevScrollY
     },
+
     isScrollingUp () {
       return this.currScrollY < this.prevScrollY
+    },
+
+    isEligibleForScroll () {
+      return this.currScrollY > this.initializeHideHeight
     }
   },
   methods: {
     fixedTopFall: _.debounce(function () {
       this.currScrollY = window.scrollY
 
-      if ((this.currScrollY > this.initializeHideHeight) && this.isScrollingDown) {
+      if (!this.mobileMenuClicked && this.isEligibleForScroll && this.isScrollingDown) {
```

Comment:
Same as bellow, mobileMenuClicked - > mobileMenuActive or mobileMenuCollapsed, because you don't know what menu state is when it is clicked.